src/components/Accordion.js

```diff
@@ -0,0 +1,93 @@
+const menuOpen = "collapse show";
+const buttonOpenAll = "Open All";
+const buttonCloseAll = "Close All";
+
+if (!String.prototype.includes) {
+  String.prototype.includes = function(search, start) {
+    "use strict";
+    if (typeof start !== "number") {
+      start = 0;
+    }
+
+    if (start + search.length > this.length) {
+      return false;
+    } else {
+      return this.indexOf(search, start) !== -1;
+    }
+  };
+}
+
+document.addEventListener(
+  "click",
+  clickEvent => {
```

Comment:
can we just move this to a named function, i suggest `onDocumentClick`

Reply:
done

Reply:
sorry, this would look something like

```js
const onDocumentClick = clickEvent => {
// function stuff goes here
};

document.addEventListener(
  "click",
  onDocumentClick);
```
```

Reply:
done